modules/govuk_elasticsearch/templates/es-backup-s3.erb

```diff
@@ -15,6 +15,19 @@ function nagios_passive () {
 
 trap nagios_passive EXIT
 
+<%- if scope.lookupvar('::aws_migration') %>
+  # Get the instance ID for the specific machine.
+  INSTANCE_ID=ec2metadata --instance-id
```

Comment:
We have the instance ID in a facter too, we could use it in the template. Anyway, the exec needs to be wrapped in $() or /` /`